jobmaster/cvsJob/cvsJobMaster.go

```diff
@@ -114,26 +115,28 @@ func (jm *JobMaster) Tick(ctx context.Context) error {
 			continue
 		}
 		status := worker.handle.Status()
-		if status.Code == lib.WorkerStatusNormal {
+		if status.Code == lib.WorkerStatusNormal || status.Code == lib.WorkerStatusFinished {
 			num, err := strconv.ParseInt(string(status.ExtBytes), 10, 64)
 			if err != nil {
 				return err
 			}
 			worker.curLoc = num
 			jm.counter += num
-			log.L().Debug("cvs job tmp num ", zap.Any("id :", worker.handle.ID()), zap.Int64("counter: ", num))
+			log.L().Debug("cvs job tmp num ", zap.Any("id", worker.handle.ID()), zap.Int64("counter", num), zap.Any("status", status.Code))
 			// todo : store the sync progress into the meta store for each file
-		} else if status.Code == lib.WorkerStatusFinished {
-			// todo : handle error case here
-			log.L().Info("sync file finished ", zap.Any("message", worker.file))
 		} else if status.Code == lib.WorkerStatusError {
 			log.L().Error("sync file failed ", zap.Any("message", worker.file))
 		} else {
 			log.L().Info("worker status abnormal", zap.Any("status", status))
 		}
 	}
 	log.L().Info("cvs job master status  ", zap.Any("id :", jm.workerID), zap.Int64("counter: ", jm.counter))
-	return nil
+	err := jm.BaseJobMaster.UpdateJobStatus(ctx, jm.Status())
```

Comment:
don't call updatejobstatus frequently, for the updateJobstatus will store the status into the megastore.   Only call it when the status changes

Reply:
fixed, PTAL

Reply:
updated, PTAL again